### PR TITLE
pciutils: fix compilation with GCC14

### PR DESCRIPTION
**Pull request by neheb**
Missing header.

Maintainer: @lucize

### CODE REVIEW

File: utils/pciutils/patches/110-gcc14.patch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--- a/lib/sysfs.c
```
Comment (BKPepe) [inline utils/pciutils/patches/110-gcc14.patch:1]: I fail to find this change in the pciutils repository. Did you even try to submit all your recent GCC14 patches to upstream? Why we should carry these patches for ages?

All patches according to OpenWrt policy should be in shape to be upstreamable. Here is missing Git header with all details like who is the author, commit subject, description, etc.

**Review by BKPepe — CHANGES_REQUESTED** [review]

File: utils/pciutils/patches/110-gcc14.patch (hunk 1 above)
Comment (neheb) [reply utils/pciutils/patches/110-gcc14.patch:1]: Some were upstreamed, yes.

**Comment by lucize** [comment]
LGTM

**Comment by httpstorm** [comment]
Build and run tested: WRT3200ACM.

**Pull request merged**